### DISCUSSION TITLE
Spektrum AR7200BX

### DISCUSSION THREAD

**Discussion by RobertoD91** [pascallanger/DIY-Multiprotocol-TX-Module#991] [Q&A] [answered]
Hello! I'm doing some troubleshooting, i have a Spektrum AR7200BX Flybarless system, according to the [manual](https://www.horizonhobby.com/on/demandware.static/Sites-horizon-us-Site/Sites-horizon-master/default/Manuals/SPMAR7200BX-Manual_EN.pdf) this rx is:

> Channels: 7 or 8
DSM2 or DSMX
Resolution: 2048
Frame Rate: 11ms

So in multimodule i have to set:
- Sub_protocol DSM2_2F or Sub_protocol DSMX_2F
- subtype normal
- servo rate 11ms
- i can leave channel number from 1 to 12

Right?

Some screenshot (sorry they are in italian):
![screen-2024-06-22-005632](https://github.com/pascallanger/DIY-Multiprotocol-TX-Module/assets/298514/0b120ded-6d02-41cc-897f-1bd7e28677d8)
![screen-2024-06-22-005641](https://github.com/pascallanger/DIY-Multiprotocol-TX-Module/assets/298514/736684e8-7b56-40fd-ac02-9a6f1b68260a)

**Comment by pascallanger** [comment, marked as answer]
Hi Roberto, why don't you let the module negotiate by setting it to DSM/auto and let it set the parameters for you?
With 12 channels you can't have 11ms servo refresh rate. I can't remember but it's 8 or 7 channels max.

**Reply by RobertoD91** [reply]
Thank you, using the Auto discovery protocol the setting are the same.
I had tried manual settings because my servos were behaving strangely, and I was checking each component (rx, fbl, ecc).
In the end, I found that the servos themselves were faulty :(
